src/Sarif.Multitool.Library/SuppressCommand.cs

```diff
@@ -0,0 +1,82 @@
+{
+    public class SuppressCommand : CommandBase
+    {
+        public SuppressCommand(IFileSystem fileSystem = null) : base(fileSystem)
+        {
+        }
+
+        public int Run(SuppressOptions options)
+        {
+            try
+            {
+                Console.WriteLine($"Suppress '{options.InputFilePath}' => '{options.OutputFilePath}'...");
+                var w = Stopwatch.StartNew();
+
+                bool valid = ValidateOptions(options);
+                if (!valid)
+                {
+                    return FAILURE;
+                }
+
+                SarifLog currentSarifLog = PrereleaseCompatibilityTransformer.UpdateToCurrentVersion(FileSystem.FileReadAllText(options.InputFilePath),
+                                                                                                     options.Formatting,
+                                                                                                     out string _);
+
+                SarifLog reformattedLog = new SuppressVisitor(options.Justification,
+                                                              options.Alias,
+                                                              options.Guids,
+                                                              options.Timestamps,
+                                                              options.ExpiryInDays,
+                                                              options.Status).VisitSarifLog(currentSarifLog);
+
+                string actualOutputPath = CommandUtilities.GetTransformedOutputFileName(options);
+                if (options.SarifOutputVersion == SarifVersion.OneZeroZero)
+                {
+                    var visitor = new SarifCurrentToVersionOneVisitor();
+                    visitor.VisitSarifLog(reformattedLog);
+
+                    WriteSarifFile(FileSystem, visitor.SarifLogVersionOne, actualOutputPath, options.Formatting, SarifContractResolverVersionOne.Instance);
+                }
+                else
+                {
+                    WriteSarifFile(FileSystem, reformattedLog, actualOutputPath, options.Formatting);
+                }
+
+                w.Stop();
+                Console.WriteLine($"Supress completed in {w.Elapsed}.");
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex);
+                return FAILURE;
+            }
+
+            return SUCCESS;
+        }
+
+        private bool ValidateOptions(SuppressOptions options)
+        {
+            bool valid = true;
+
+            valid &= options.Validate();
+            valid &= options.ExpiryInDays >= 0;
+            valid &= !string.IsNullOrWhiteSpace(options.Justification);
+            valid &= (options.Status == SuppressionStatus.Accepted || options.Status == SuppressionStatus.UnderReview);
+            valid &= DriverUtilities.ReportWhetherOutputFileCanBeCreated(options.OutputFilePath, options.Force, FileSystem);
```

Comment:
Even if every other item is false, this will return true if the last line is true, is this intended behavior?

Conversely, if every other item is true, this will return false if the last item is false. I can see why that might be useful in this case, but again, is that intended behavior?